app/views/TeamChannelsView.tsx

```diff
@@ -435,7 +436,8 @@ class TeamChannelsView extends React.Component<ITeamChannelsViewProps, ITeamChan
 				{
 					text: I18n.t('Yes_action_it', { action: I18n.t('delete') }),
 					style: 'destructive',
-					onPress: () => deleteRoom(item._id, item.t)
+					// VERIFY ON PR
+					onPress: () => dispatch(deleteRoom(item._id, item))
```

Comment:
We need to address this and open a new task with proper info.
Or is it easy enough to fix on this PR?

Reply:
Actually, I think it doesn't matter, but as I said it was already broken in the previous version, because everywhere they instantiate the return object and not the string that was passed.